framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1588,6 +1615,10 @@ protected void applyInferredAnnotations(AnnotatedTypeMirror type, Value as) {
     /**
      * Applies defaults for types in a class with an qualifier parameter.
      *
+     * <p>Within a class with {@code @HasQualifierParameter}, types with that class default to the
+     * polymorphic qualifier rather than the typical default. Local variables with a type that has a
+     * qualifier parameter are initialized to the type of their initializer, rather than top.
```

Comment:
Maybe instead of `top` say the default for local variables, as a type system could change that default.